README.md

```diff
@@ -1,2 +1,53 @@
 # more-recipes
 For the foodies in the house
+
+[![Build Status](https://travis-ci.org/Merdoth/more-recipes.svg?branch=develop)](https://travis-ci.org/Merdoth/more-recipes)
+[![Coverage Status](https://coveralls.io/repos/github/Merdoth/more-recipes/badge.svg?branch=develop)](https://coveralls.io/github/Merdoth/more-recipes?branch=develop)
+
+
+More recipes is a web appliaction that allows food lovers to exhibit and explore their ideas towards food, and gets the chance to share with others what they've been able to come across in the course of exploring. It features are:
```

Comment:
`Nitpick`: I believe the last sentence should've been `Its features are: . . .` or `It features: . . .`